[debug] [2025-09-23T14:34:07.655Z] ----------------------------------------------------------------------
[debug] [2025-09-23T14:34:07.658Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Alejandro\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2025-09-23T14:34:07.658Z] CLI Version:   14.17.0
[debug] [2025-09-23T14:34:07.658Z] Platform:      win32
[debug] [2025-09-23T14:34:07.658Z] Node Version:  v22.18.0
[debug] [2025-09-23T14:34:07.659Z] Time:          Tue Sep 23 2025 11:34:07 GMT-0300 (hora estándar de Argentina)
[debug] [2025-09-23T14:34:07.659Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-09-23T14:34:07.661Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  The Firebase CLI’s MCP server feature can optionally make use of Gemini in Firebase. Learn more about Gemini in Firebase and how it uses your data: https://firebase.google.com/docs/gemini-in-firebase#how-gemini-in-firebase-uses-your-data 
[info] 
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you. 
[info] 
[info] i  To change your preferences at any time, run `firebase logout` and `firebase login` again. 
[info] 
[info] Visit this URL on this device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=112241181&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2025-09-23T14:34:29.710Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: 
[warn] !  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2025-09-23T14:34:29.712Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2025-09-23T14:34:29.749Z] >>> [apiv2][query] POST https://accounts.google.com/o/oauth2/token [none]
[debug] [2025-09-23T14:34:29.749Z] >>> [apiv2][body] POST https://accounts.google.com/o/oauth2/token [omitted]
[debug] [2025-09-23T14:34:50.814Z] *** [apiv2] error from fetch(https://accounts.google.com/o/oauth2/token, {"headers":{},"method":"POST","body":{"_overheadLength":543,"_valueLength":208,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------3173365488ece1dfa0bf9d9c","_events":{},"_eventsCount":1}}): FetchError: request to https://accounts.google.com/o/oauth2/token failed, reason: connect ETIMEDOUT 142.251.0.84:443
[debug] [2025-09-23T14:34:50.815Z] Token Fetch Error: FirebaseError: Failed to make request to https://accounts.google.com/o/oauth2/token
    at RetryOperation._fn (C:\Users\Alejandro\AppData\Roaming\npm\node_modules\firebase-tools\lib\apiv2.js:261:31)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
[error] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-09-23T14:34:50.826Z] >>> [apiv2][query] POST https://auth.firebase.tools/attest [none]
[debug] [2025-09-23T14:34:50.826Z] >>> [apiv2][body] POST https://auth.firebase.tools/attest {"session_id":"ed6079bb-ba33-4edb-8c58-2f2370512589"}
[debug] [2025-09-23T14:35:12.643Z] *** [apiv2] error from fetch(https://auth.firebase.tools/attest, {"headers":{},"method":"POST","body":"{\"session_id\":\"ed6079bb-ba33-4edb-8c58-2f2370512589\"}"}): FetchError: request to https://auth.firebase.tools/attest failed, reason: 
[debug] [2025-09-23T14:35:17.661Z] Error: Timed out.
    at Timeout._onTimeout (C:\Users\Alejandro\AppData\Roaming\npm\node_modules\firebase-tools\lib\utils.js:275:49)
    at listOnTimeout (node:internal/timers:588:17)
    at process.processTimers (node:internal/timers:523:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2025-09-23T14:55:50.038Z] ----------------------------------------------------------------------
[debug] [2025-09-23T14:55:50.041Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Alejandro\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js deploy --only hosting --project gestionprestamosinformaticos --non-interactive
[debug] [2025-09-23T14:55:50.042Z] CLI Version:   14.17.0
[debug] [2025-09-23T14:55:50.042Z] Platform:      win32
[debug] [2025-09-23T14:55:50.042Z] Node Version:  v22.18.0
[debug] [2025-09-23T14:55:50.043Z] Time:          Tue Sep 23 2025 11:55:50 GMT-0300 (hora estándar de Argentina)
[debug] [2025-09-23T14:55:50.043Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-09-23T14:55:50.044Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2025-09-23T14:55:50.253Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-23T14:55:50.253Z] > authorizing via FIREBASE_TOKEN environment variable
[warn] !  Authenticating with `FIREBASE_TOKEN` is deprecated and will be removed in a future major version of `firebase-tools`. Instead, use a service account key with `GOOGLE_APPLICATION_CREDENTIALS`: https://cloud.google.com/docs/authentication/getting-started 
[debug] [2025-09-23T14:55:50.253Z] [iam] checking project gestionprestamosinformaticos for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2025-09-23T14:55:50.254Z] No OAuth tokens found
[debug] [2025-09-23T14:55:50.254Z] No OAuth tokens found
[debug] [2025-09-23T14:55:50.254Z] > refreshing access token with scopes: []
[debug] [2025-09-23T14:55:50.255Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-09-23T14:55:50.255Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-23T14:56:12.065Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: 
[warn] !  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2025-09-23T14:56:12.066Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2025-09-23T14:56:14.408Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":149,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------51bd170c900b0ba82832e481","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: 
[error] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-09-23T14:56:14.410Z] [iam] error while checking permissions, command may fail: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-09-23T14:56:14.413Z] No OAuth tokens found
[debug] [2025-09-23T14:56:14.413Z] No OAuth tokens found
[debug] [2025-09-23T14:56:14.413Z] > refreshing access token with scopes: []
[debug] [2025-09-23T14:56:14.414Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-09-23T14:56:14.414Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-23T14:56:38.510Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":149,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------ee63aa5b42f68c8bd4fb2aa9","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: 
[error] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-09-23T14:56:38.511Z] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-09-23T14:56:43.516Z] Error: Timed out.
    at Timeout._onTimeout (C:\Users\Alejandro\AppData\Roaming\npm\node_modules\firebase-tools\lib\utils.js:275:49)
    at listOnTimeout (node:internal/timers:588:17)
    at process.processTimers (node:internal/timers:523:7)
[error] 
[error] Error: An unexpected error has occurred.
